tasks/base.py

```diff
@@ -35,16 +36,22 @@
 )
 
 
+def _get_task_group_from_name(task_name: Optional[str]) -> Optional[str]:
+    if task_name is None or ("." not in task_name):
+        return None
```

Comment:
[nit] do we want this to return `None` or `"unknown_group"`

Reply:
At first I thought "unknown_group" but after checking the existing dashboards for the tasks I noticed that it's possible to have the task_name there are `None`... so just being consistent with returning `None` as well